app/helpers/view_helper.rb

```diff
@@ -22,4 +22,28 @@ def search_ui_url(relative_path)
   def search_ui_course_page_url(provider_code:, course_code:)
     search_ui_url("/course/#{provider_code}/#{course_code}")
   end
+
+  def course_has_unpublished_changes(course)
```

Comment:
Both of these methods are specific to courses. How about creating a new `helpers/course_helper.rb` for them to live in?